test/baselines/generated-reference/autolinking.html

```diff
@@ -8,7 +8,7 @@
 </ul></div><div id="spec-container">
 <emu-clause id="sec-foo">
   <h1><span class="secnum">1</span> Autolinking</h1>
-  <p>Type, type, <emu-xref aoid="Type" id="_ref_0"><a href="https://tc39.es/ecma262/#sec-ecmascript-data-types-and-values">Type</a></emu-xref>(), type()</p>
+  <p>Type, type, <emu-xref aoid="Type"><a href="https://tc39.es/ecma262/#sec-ecmascript-data-types-and-values">Type</a></emu-xref>(), type()</p>
```

Comment:
This change was due to the "move logic for registering op biblio entries" commit. 

Background: For `xref`s to terms defined in the current document, we add an `id` to the `xref` so we can link to it from the References pane when hovering the definition of the term. That's what the `id="_ref_0"` attributes are.

Prior to the referenced commit, we created the entry for an AO as part of adding the entry for a clause; that meant that when loading an external biblio containing an entry for a clause, we'd re-recreate the entry for its AO (thus shadowing the original one, from the external biblio). But all created newly-entries are considered to be in the _current_ namespace, so the analysis thought the AO was defined in the current document, and hence `xref`s to it should be given `id`s. 

The tweak in that commit means we don't re-create the biblio entry for the Type AO, and just use the original, external one. As a consequence, the analysis correctly recognizes that this xref is to an external term and so does not need to be given an `id`. Hence the diff.